### ISSUE TITLE
Got private method `password' called for "xxxxxxxx":String (NoMethodError) when using distribute:itunesconnect

### ISSUE THREAD

**Issue by Sushisugre** [nomad/shenzhen#195]
I got the following error when using -p flag, or ITUNES_CONNECT_PASSWORD in environment variable  

```
private method `password' called for "xxxxxxxx":String (NoMethodError) when using -p or ITUNES_CONNECT_PASSWORD in environment virable 
```
Here's the command and stacktrace:

```
$ ipa distribute:itunesconnect -a $ACCOUNT -p $PASSWORD -i $APPLICATION_ID --trace
/Library/Ruby/Gems/2.0.0/gems/shenzhen-0.10.3/lib/shenzhen/plugins/itunesconnect.rb:109:in `block (2 levels) in <top (required)>': private method `password' called for "xxxxxxxx":String (NoMethodError)
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.2.1/lib/commander/command.rb:180:in `call'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.2.1/lib/commander/command.rb:180:in `call'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.2.1/lib/commander/command.rb:155:in `run'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.2.1/lib/commander/runner.rb:421:in `run_active_command'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.2.1/lib/commander/runner.rb:81:in `run!'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.2.1/lib/commander/delegates.rb:8:in `run!'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.2.1/lib/commander/import.rb:10:in `block in <top (required)>'
```

**Comment by KrauseFx** [comment]
This is related to this pull request: https://github.com/nomad/shenzhen/pull/181 which is merged, but not yet released. 
If you use the latest version from the master branch you should be good to go :+1:

**Comment by mattt** [comment]
Fixed for version 0.11.0.
